env_setup/env_setup.yml

```diff
@@ -67,8 +64,41 @@
         esxi_shell: "/bin/supershell"
       when: esxi_version is version('8.0.0', '>=')
 
-    # Add esxi host into memory inventory
-    - include_tasks: ../common/add_host_in_memory_inventory.yml
+    - name: "Cleanup snapshots and get VM related information"
+      block:
+        - name: "Not cleanup old snapshot by default"
+          ansible.builtin.set_fact:
+            cleanup_old_snapshots: false
+          when: cleanup_old_snapshots is undefined
+
+        - name: "Cleanup existing snapshots of existing VM if required"
```

Comment:
Change name to "Revert and cleanup existing snapshots on existing VM if required"

Reply:
Thanks. change it.